src/client/public/config/openfin/platforms.json

```diff
@@ -0,0 +1,191 @@
+                                                    "componentState": {
+                                                        "url": "http://localhost:3000/tiles",
+                                                        "name": "Tiles"
+                                                    },
+                                                    "isClosable": true,
+                                                    "reorderEnabled": true,
+                                                    "title": "view"
+                                                }
+                                            ]
+                                        },
+                                        {
+                                            "type": "stack",
+                                            "header": {},
+                                            "isClosable": true,
+                                            "reorderEnabled": true,
+                                            "title": "",
+                                            "activeItemIndex": 0,
+                                            "height": 50,
+                                            "content": [
+                                                {
+                                                    "type": "component",
+                                                    "componentName": "view",
+                                                    "componentState": {
+                                                        "url": "http://localhost:3000/blotter",
+                                                        "name": "Blotter"
+                                                    },
+                                                    "isClosable": true,
+                                                    "reorderEnabled": true,
+                                                    "title": "view"
+                                                }
+                                            ]
+                                        }
+                                    ]
+                                },
+                                {
+                                    "type": "stack",
+                                    "width": 30,
+                                    "height": 100,
+                                    "isClosable": true,
+                                    "reorderEnabled": true,
+                                    "title": "",
+                                    "activeItemIndex": 0,
+                                    "content": [
+                                        {
+                                            "type": "component",
+                                            "componentName": "view",
+                                            "componentState": {
+                                                "url": "http://localhost:3000/analytics",
```

Comment:
Tab headers need to each be set according to their route. Will need to use something like `react-helmet`.